packages/cli/src/cmds/archive/index.ts

```diff
@@ -39,7 +39,14 @@ export async function index(
   );
 
   if (sampleMetadata) {
-    await emitUsage(appMapDir, totalEvents, result.numProcessed, sampleMetadata);
+    const outputDir = join('.appmap', 'run-stats');
```

Comment:
Is this meant to go in the cwd?
```suggestion
    const outputDir = join(homedir(), '.appmap', 'run-stats');
```

Reply:
It actually is meant to go in the `cwd`, because this command is designed for use in CI. The data file gets written to `.appmap/run-stats` where it gets picked up by the api services backend.

You can see the old implementation was:

```
const runStatsDirectory = join('.appmap', 'run-stats');
```

So that directory is now an input parameter to `writeUsage`.